app/models/course/material.rb

```diff
@@ -2,8 +2,28 @@
 class Course::Material < ApplicationRecord
   has_one_attachment
   include DuplicationStateTrackingConcern
+  include Workflow
```

Comment:
Nit: Update of models 4d2ce4b7c9a04cfd761c7389e70e31f651fc7176 can squash with adding of associated tables https://github.com/Coursemology/coursemology2/commit/e17e840d2185a9ae59d26d0e3a1345de1352be77

This commit 4d2ce4b7c9a04cfd761c7389e70e31f651fc7176 should also come before `build_text_chunks` is used in 07d05fbd4f11a0a79a0adcc463c7858987ca6f2f

Reply:
Reordered

Reply:
Squashed